egs2/chime4/enh_asr1/run.sh

```diff
@@ -41,5 +41,5 @@ word_vocab_size=65000
     --train_set "${train_set}" \
     --valid_set "${valid_set}" \
     --test_sets "${test_sets}" \
-    --bpe_train_text "data/${train_set}/text" \
-    --lm_train_text "data/${train_set}/text data/local/other_text/text" "$@"
+    --bpe_train_text "data/${train_set}/text_spk1" \
```

Comment:
I think we can keep `text` here to be the same as chime4/asr1/run.sh. But either is fine.

Reply:
Yes, I made a symbolic link in `local/data.sh` for chime4. Here I just want to explicitly use `text_spk1` to reminder users. Because for multi-speaker data, the use of `text` could cause errors.